clinica/iotools/converters/oasis_to_bids/oasis_to_bids_utils.py

```diff
@@ -105,3 +104,33 @@ def write_sessions_tsv(bids_dir: Path, sessions_dict: dict) -> None:
             sep="\t",
             encoding="utf8",
         )
+
+
+def write_scans(
```

Comment:
Did you change the name from `write_scans_tsv` to `write_scans` on purpose ?

With `write_scans` you could expect the function to write more than the TSV file, which is why I'd vote for a more specific name.

WDYT ?